v3/src/data-interactive/handlers/selection-list-handler.ts

```diff
@@ -0,0 +1,50 @@
+import { IDataSet } from "../../models/data/data-set"
+import { t } from "../../utilities/translation/translate"
+import { registerDIHandler } from "../data-interactive-handler"
+import { DIHandler, DIResources, DIValues } from "../data-interactive-types"
+
+const unknownDataContextResult =
+  { success: false, values: { error: t("V3.DI.Error.dataContextUnknown")} } as const
```

Comment:
Couldn't we just use `V3.DI.Error.dataContextNotFound` here?

Reply:
I was staying true to v2 here, but I agree that it would be better to be consistent with error messages. I'll go ahead and switch to `dataContextNotFound`.